pilot/pkg/security/authz/builder/extauthz.go

```diff
@@ -85,6 +85,13 @@ func processExtensionProvider(in *plugin.InputParams) map[string]*builtExtAuthz
 			parsed, err = buildExtAuthzHTTP(in, p.EnvoyExtAuthzHttp)
 		case *meshconfig.MeshConfig_ExtensionProvider_EnvoyExtAuthzGrpc:
 			parsed, err = buildExtAuthzGRPC(in, p.EnvoyExtAuthzGrpc)
+		case *meshconfig.MeshConfig_ExtensionProvider_Datadog, *meshconfig.MeshConfig_ExtensionProvider_Lightstep,
```

Comment:
I think you can just add the continue to the `default`, we don't need to check unsupported extension provider here.

Reply:
sg. done.